kolibri/core/templates/kolibri/base.html

```diff
@@ -10,15 +10,42 @@
   <title>{% trans "Kolibri" %}</title>
 </head>
 <style>
-  .loading-spinner {
+  .loading-spinner-wrapper-base{
     width: 97vw;
     height: 97vh;
+    position: relative;
+    opacity: 0;
+    animation-duration: 0s;
+    animation-delay: 250ms;
+    animation-name: delaydisplay;
+    animation-fill-mode: forwards;
+  }
+
+  @keyframes delaydisplay {
+    from {
+      opacity: 0;
+    }
+    to {
+      opacity: 1;
+    }
+  }
+
+  .loading-spinner-base {
+    width: 50%;
+    height: 50%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
     background: url("{% static 'images/loading-spinner.gif' %}") no-repeat center;
+    background-size: contain;
```

Comment:
Looks like these styles and classes are only meant to take effect before the vue has been loaded?

In that case, it might be meant to move the style block into the `<rootvue>` tag below, so they are destroyed once the vue loads.